subt/docker/robotika/ros/robotika/src/drone_height.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/python
+
+from __future__ import print_function
+import rospy
+from nav_msgs.msg import *
+from std_msgs.msg import *
+from sensor_msgs.msg import *
+from geometry_msgs.msg import *
+
+import pdb
+    
+HEIGHT = 1.5
+MAX_ANGULAR = 0.7
+lastScanDown = 0.0
+lastScanUp = 0.0
+lastOdom = None
+validOdom = True
```

Comment:
(not a strong opinion)

Could this all be wrapped in an object? In the current form, the whole module is essentially a singleton and cannot be used twice in the same address space. On the other hand, I am not sure if more instances are needed and if the rospy.Subscriber call bellow becomes a lot harder with objects.

Reply:
Quite contrary: using a bound method for `rospy.Subsriber` actually makes everything simpler.